js/config.js

```diff
@@ -18,117 +18,55 @@ if (config.audioUrl && !config.initialTracks) {
 export const skinUrl = config.skinUrl === undefined ? null : config.skinUrl;
 export const initialTracks = config.initialTracks || [
   {
-    metaData: { artist: "DJ Mike Llama", title: "Llama Whippin' Intro" },
-    url: llamaAudio,
-    duration: 5.322286
```

Comment:
Let's actually keep these metadata/duration. Without them we would always have to load music-metadata and fetch (at least part of) each media file, and we can show the proper title/duration in the playlist immediately. With them, we can defer loading music-metadata until someone drags in an unknown track.

Reply:
Sure. It's done on isolated commit (472c2c0a010961d8cc35e3bd736455699b8df83f) on purpose, let's revert that one at the for the sake of easy testing until we merge back this branch.